src/ui/common/src/components/MultiFileViewer/index.tsx

```diff
@@ -90,22 +100,25 @@ const MultiFileViewer: React.FC<Props> = ({ files, codeHeight = '30vh' }) => {
           </TreeItem>
         );
       });
+
       return [...folderItems, ...fileItems];
     }
   };
 
-  const options = { readOnly: true, minimap: { enabled: false } };
-  if (matches) {
-    options.minimap.enabled = true;
-  }
-
+  const options = {
+    readOnly: true,
+    minimap: { enabled: false },
+    wordWrap: 'on' as 'on' | 'off' | 'wordWrapColumn' | 'bounded',
```

Comment:
Is there a particular reason why we can't just write `wordWrap: 'on'`?

Reply:
There's something weird going on with the Typescript typechecker here that I can't quite figure out. The type of the `wordWrap` argument is set to `'on' | 'off' | 'wordWrapColumn' | 'bounded'`, and when I just pass in `'on'`, it treats it as type `string`. 

The only way I could get the compiler to be happy was by forcing the type like this.